### ISSUE TITLE
getInputProps returning empty object, despite passing inputProps to renderFrames

### ISSUE THREAD

**Issue by natasha-93** [JonnyBurger/remotion#313]
I'm trying to pass in some input props on the server like this:

```
const {assetsInfo} = await renderFrames({
  config: compMetadata,
  webpackBundle: bundled,
  onStart: () => console.log('Rendering frames...'),
  onFrameUpdate: (f) => {
    if (f % 10 === 0) {
      console.log(`Rendered frame ${f}`);
    }
  },
  parallelism: null,
  outputDir: tmpDir,
  inputProps: {
    test: {
      foo: 'Bar',
    },
  },
  compositionId,
  imageFormat,
});
```

But they're not accessible via `getInputProps` in my composition/root:

```
export const RemotionVideo = () => {
  const inputProps = getInputProps();
  
  // This error is being thrown
  if (Object.values(inputProps).length === 0) {
    throw new Error('missing');
  }
  
  return (
    <>
      <Composition
        id="Project"
        component={Project}
        durationInFrames={100}
        fps={24}
        width={640}
        height={360}
      />
    </>
  );
};
```

**Comment by JonnyBurger** [comment]
Thanks for reporting this!
I was able to reproduce this and started working on a fix #314.

I'll test it and release it this evening or by tomorrow latest!

**Comment by JonnyBurger** [comment]
There is a bugfix out now with version 2.0.6 of Remotion `npm run upgrade`.

You need to make 1 change to your code: Add the `inputProps` also to your `getCompositions()` call:

```tsx
const compositions = await getCompositions(bundled, {
  inputProps: {
    test: {
      foo: 'Bar',
    },
  }
})
```

This was poorly documented and the extra `getCompositions()` parameter was not in the example or default template, so I just shipped an update to the template and documented the whole SSR flow much better, so `getCompositions()` [for example now also has a documentation page](https://www.remotion.dev/docs/get-compositions), so hopefully there will be less struggles along the way!
